outputs.tf

```diff
@@ -2,7 +2,24 @@
 # Outputs
 ########################################################################################################################
 
-#output "myoutput" {
-#  description = "Description of my output"
-#  value       = "value"
-#}
+output "name" {
+  description = "Name of created SCC WP instance."
+  value       = ibm_resource_instance.scc_wp.name
+}
+
+output "id" {
+  description = "ID of created SCC WP instance."
+  value       = ibm_resource_instance.scc_wp.crn
+}
+
+output "sysdig_collector_endpoint" {
```

Comment:
ingest_endpoint

Reply:
used ingestion_endpoint since we already mention it inside observability agent module